### DISCUSSION TITLE
Form error

### DISCUSSION THREAD

**Discussion by Nuel-Eneji** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#9] [Outreachy 2022] [answered]
Salutations to @alswajiab and @BatoolMM  I'm a Nigerian outreachy internship candidate named Nuel Eneji. I'm so happy to be here and would genuinely love to be a part of this community, but I've been having the same problem as the other applicants and am eagerly awaiting a fix.
Thanks

**Comment by BatoolMM** [comment, marked as answer]
Hi @Nuel-Eneji, 
Apologies for the broken link, is [this one](https://form.jotform.com/221813832742353) working for you?

**Reply by Nuel-Eneji** [reply]
Thank you for the link; indeed, it operates flawlessly. I had already signed up straight on the website, but I haven't yet located the community.
Having completed the form, I received the success message seen in the screenshot below.
![Screenshot from 2022-10-09 08-11-12](https://user-images.githubusercontent.com/100046756/194743111-47f19ff7-dbfb-4eda-a6bb-1e3aa1686dfb.png)

**Reply by BatoolMM** [reply]
Glad it worked, the website of the community is https://osc-ksa.com/. Please feel free to open a PR with your contribution in the GitHub repository!

**Reply by Nuel-Eneji** [reply]
Thanks for the help @BatoolMM, I'm looking forward to working with you

**Comment by Nuel-Eneji** [comment]
Greetings, @BatoolMM I hope all is okay with you. I'm having some trouble locating the projects to which I can donate.
I am a frontend developer with the necessary knowledge of HTML, CSS, and JavaScript.
